packages/SwingSet/misc-tools/store-stats/src/index.js

```diff
@@ -0,0 +1,43 @@
+import processAmbient from 'process';
+import dbOpenAmbient from 'better-sqlite3';
+import {
+  makeStatsSender,
+  makeStatsTransformer,
+  makeSwingstore,
+} from './store-stats.js';
+
+export const swingstorePath = '~/.agoric/data/agoric/swingstore.sqlite';
+
+/**
+ * @param {string[]} argv
+ * @param {{ dbOpen: typeof import('better-sqlite3'), HOME?: string }} io
+ */
+const main = async (argv, { dbOpen }) => {
+  const [_node, _script, databasePath] = argv;
+  const fullPath = (databasePath || swingstorePath).replace(
+    /^~/,
+    processAmbient.env.HOME,
+  );
+  console.log(`Opening database at ${fullPath}`);
+  const kStore = makeSwingstore(dbOpen(fullPath, { readonly: true }));
+  const height = kStore.findHeight();
+  const vatItems = kStore.findVatItems();
```

Comment:
maybe named this `vatItemIterator` to make it more obvious that we aren't reading the entire DB into RAM first